client-js/evomaster-client-js/src/instrumentation/babel-plugin-evomaster.ts

```diff
@@ -153,6 +153,20 @@ export default function evomasterPlugin(
         branchCounter++;
     }
 
+    function doesContainExpression(node: Expression) : boolean{
```

Comment:
can u add a comment to explain it? i mean, does it check if something is `asyn`? (and if yes, Asyn should be somehow in the name)